src/Mod/CAM/Path/Preferences.py

```diff
@@ -114,9 +114,11 @@ def getBuiltinToolBitPath() -> pathlib.Path:
     return getBuiltinAssetPath() / "Bit"
 
 
-def getDefaultAssetPath():
-    config = pathlib.Path(FreeCAD.ConfigGet("UserConfigPath"))
-    return config / "CamAssets"
+def getDefaultAssetPath() -> Path:
+    data_dir = Path(FreeCAD.getUserAppDataDir())
```

Comment:
This looks like it needs to be 
```suggestion
    data_dir = pathlib.Path(FreeCAD.getUserAppDataDir())
```